ui/components/User.js

```diff
@@ -160,6 +161,21 @@ const User = (props) => {
   const { color, iconButtonClassName, avatarClassName, classes } = props;
 
   const open = Boolean(anchorEl);
+
+
+  if ( userData?.status == 'anonymous' ) {
+    const url =
+         `${capabilitiesRegistry?.provider_url}/user/login?anonymousUserID=${userData?.user_id}&redirect=${window.location.pathname}`
+        
+    return (
```

Comment:
It'd be great to have a small section in https://docs.meshery.io/extensibility/ui explain the purpose of this extension point.